project/AppDependencies.scala

```diff
@@ -7,28 +7,28 @@ object AppDependencies {
   val compile = Seq(
     ws,
     "uk.gov.hmrc"          %% "play-conditional-form-mapping" % "1.3.0-play-26",
-    "uk.gov.hmrc"          %% "logback-json-logger"           % "4.8.0",
-    "uk.gov.hmrc"          %% "govuk-template"                % "5.55.0-play-26",
-    "uk.gov.hmrc"          %% "play-health"                   % "3.15.0-play-26",
-    "uk.gov.hmrc"          %% "play-ui"                       % "8.14.0-play-26",
-    "uk.gov.hmrc"          %% "bootstrap-frontend-play-26"    % "2.24.0",
-    "uk.gov.hmrc"          %% "play-frontend-govuk"           % "0.53.0-play-26",
-    "uk.gov.hmrc"          %% "play-frontend-hmrc"            % "0.26.0-play-26",
+    "uk.gov.hmrc"          %% "logback-json-logger"           % "4.9.0",
+    "uk.gov.hmrc"          %% "govuk-template"                % "5.60.0-play-27",
+    "uk.gov.hmrc"          %% "play-health"                   % "3.16.0-play-27",
+    "uk.gov.hmrc"          %% "play-ui"                       % "8.19.0-play-27",
+    "uk.gov.hmrc"          %% "bootstrap-frontend-play-27"    % "3.2.0",
+    "uk.gov.hmrc"          %% "play-frontend-govuk"           % "0.56.0-play-27",
+    "uk.gov.hmrc"          %% "play-frontend-hmrc"            % "0.34.0-play-27",
     "ai.x"                 %% "play-json-extensions"          % "0.42.0",
-    "uk.gov.hmrc"          %% "play-whitelist-filter"         % "3.4.0-play-26",
-    "uk.gov.hmrc"          %% "simple-reactivemongo"          % "7.30.0-play-26",
-    "uk.gov.hmrc"          %% "play-json-union-formatter"     % "1.10.0-play-26",
+    "uk.gov.hmrc"          %% "play-whitelist-filter"         % "3.4.0-play-27",
+    "uk.gov.hmrc"          %% "simple-reactivemongo"          % "7.31.0-play-27",
+    "uk.gov.hmrc"          %% "play-json-union-formatter"     % "1.12.0-play-27",
     "com.github.tototoshi" %% "scala-csv"                     % "1.3.6",
     "com.github.cloudyrock.mongock"  %  "mongock-core"        % "2.0.2",
     "org.mongodb"          %  "mongo-java-driver"             % "3.12.1",
-    "org.webjars.npm"      %  "govuk-frontend"                % "3.9.1",
-    "org.webjars.npm"      %  "hmrc-frontend"                 % "1.20.0",
+    "org.webjars.npm"      %  "govuk-frontend"                % "3.10.2",
+    "org.webjars.npm"      %  "hmrc-frontend"                 % "1.23.1",
     "org.webjars.npm"      %  "accessible-autocomplete"       % "2.0.3"
   )
 
   val test = Seq(
     "org.scalatest"          %% "scalatest"          % "3.0.8"             % "test, it",
```

Comment:
3.2.3

Reply:
It doesn't seem possible to use 3.2.3 of scalatest as there are some incompatibilities between this, the version of play and scalatestplus-play, so I've gone back to 3.0.8

Reply:
Indeed. That's because in +3.2.0 a few Scalatest classes were deprecated (e.g. `org.scalatest.WordSpec` and `org.scalatest.MustMatchers`) and replaced by new ones (`org.scalatest.wordspec.AnyWordSpec` and `org.scalatest.matchers.should.Matchers`).

But I understand that this change would be probably challenging.

We can maybe have a quick chat with the other devs to discuss if it's worth to complete the task with this ticket or open a new one, because sooner or later the upgrade will have to be done.

Reply:
I changed all of the WordSpec and MustMatchers classes to the new ones which solved most of the errors, however there were still a couple from other dependencies which use WordSpec, I think GuiceOneAppPerSuite. I had a chat with @caoilte and it looked like they were incompatible